config/crd.yml

```diff
@@ -37,40 +37,6 @@ spec:
 ---
 apiVersion: apiextensions.k8s.io/v1beta1
 kind: CustomResourceDefinition
-metadata:
```

Comment:
Not following here on why Packages is removed. Mind explaining to me?

Reply:
CRDs are managed by the kubernetes api, so if we left package in as a CRD, it would conflict with the group+version we are trying to register our api server with.